### PR TITLE
Add netsnmp build deps

### PR DESCRIPTION
**Pull request by brian-brazil**
This will be needed over in the snmp exporter

@sdurrheimer

### CODE REVIEW

File: 1.7/base/Dockerfile [hunk 1]
```diff
@@ -8,6 +8,8 @@ RUN \
         git \
         gnupg \
         make \
+        build-essential \
+        libnetsnmp-dev \
```
Comment (sdurrheimer) [inline 1.7/base/Dockerfile]: https://packages.debian.org/fr/sid/libsnmp-dev ?

Comment (brian-brazil) [inline 1.7/base/Dockerfile]: The naming of this package has always thrown me off

File: 1.7/base/Dockerfile [hunk 2]
```diff
@@ -8,6 +8,8 @@ RUN \
         git \
         gnupg \
         make \
+        build-essential \
+        libsnmp-dev \
```
Comment (sdurrheimer) [inline 1.7/base/Dockerfile]: nits: can you order them alphabetically ?

Comment (brian-brazil) [inline 1.7/base/Dockerfile]: Done

**Pull request merged**